### PR TITLE
fix: add additonal params to build summary

### PR DESCRIPTION
**Pull request by deepankarm**

### CODE REVIEW

File: jina/docker/hubio.py [hunk 1]
```diff
@@ -264,6 +272,7 @@ def build(self) -> Dict:
             result = {
                 'name': getattr(self, 'canonical_name', ''),
                 'path': self.args.path,
+                'host_info': _host_info,
```
Comment (hanxiao) [inline jina/docker/hubio.py:275]:
```suggestion
                'host_info': _host_info if args.host_info else ''
```

Comment (hanxiao) [reply jina/docker/hubio.py:275]: add a new arg in hub build CLI `--host-info (action=store_true)`

Comment (deepankarm) [reply jina/docker/hubio.py:279]: Done

File: jina/docker/hubio.py [hunk 2]
```diff
@@ -256,6 +256,18 @@ def build(self) -> Dict:
...
+                    info, env_info = get_full_version()
+                    _host_info = {
+                        'jina': info,
+                        'jina_envs': env_info,
+                        'docker': self._raw_client.info(),
+                        'build_args': vars(self.args)
+                    }
+                else:
+                    _host_info = ''
+                    self.logger.debug('Skipping writing host_info to database')
```
Comment (hanxiao) [inline jina/docker/hubio.py:270]: `_host_info =  ''` should be added to L188 to ensure this var is always `defined` regardless `is_build_succcess`

Comment (deepankarm) [reply jina/docker/hubio.py]: This is fixed.

File: jina/docker/hubio.py [hunk 3]
```diff
@@ -256,23 +256,54 @@ def build(self) -> Dict:
...
 
             if self.args.prune_images:
                 self.logger.info('deleting unused images')
                 self._raw_client.prune_images()
 
             result = {
                 'name': getattr(self, 'canonical_name', ''),
+                'version': _version,
                 'path': self.args.path,
+                'manifest_info': _manifest_info,
```
Comment (hanxiao) [inline jina/docker/hubio.py:300]:
```suggestion
                'manifest_info': self.manifest,
```

**Review by hanxiao — APPROVED** [review]

**Pull request merged**